src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt

```diff
@@ -194,7 +194,8 @@ class Cell(
       addSpecialistCellType(it, userOrSystemInContext, clock)
     }
 
-    if (accommodationType == AccommodationType.NORMAL_ACCOMMODATION) {
```

Comment:
Are you changing the allowed enum types in this PR or another one?

Reply:
hi @Mjwillis just. I just changed the the condition. A) I changed the allowed enum using 'in' in this PR this piece of code is old **however** I haven't work the OTHER in the API in this code. (just I have seen the comment from you right now.

Reply:
@Mjwillis @danbenton-mojdt 
+ I have prevent API to accept OTHER.
+ I have not modified ENUM AccommodationType due some transaction/changes in other places else but maybe it is the right way to .